src/GroupManager.php

```diff
@@ -93,6 +101,10 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->moduleHandler->invokeAll('og_group_created', [$entity_type_id, $bundle_id]);
```

Comment:
but to circle back to @chx 's first comment - why do we need this hook to begin with?